scripts/fast-install.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
```

Comment:
Add a top-level comment to explain the purpose of the script and why/when it's needed.